server/channels/db/migrations/postgres/000122_preferences_value_length.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Preferences ALTER COLUMN TYPE text;
```

Comment:
I tested this locally with 10 million rows in the table, and it took around 5ms. I've got 1.5k preferences on Community, so that table size would represent around 7000 long time users. Even if we 20x that to get close to the 150k user benchmark we've talked about before, that should still be only 100ms which seems totally acceptable for a DB migration to take.

I also found that, while we caution against changing column types in our DB migration docs, Postgres hasn't required a table rewrite to do that since 9.2, and that's why this operation is quick